azure/requests.yaml

```
@@ -34,3 +32,7 @@ releases:
           issue: https://github.com/giantswarm/giantswarm/issues/13671
         - name: kubernetes
           version: ">= 1.17.13"
+        - name: app-operator
+          version: ">= 2.3.2"
```

```suggestion
          version: ">= 2.3.3"
```

We shipped 2.3.3 yesterday which has a fix for this PM. https://github.com/giantswarm/giantswarm/issues/13648

Our next release should only affect control planes. So I hope we don't need to update this again.

---

Thanks, I added it together with the link to PM.